reference/forms/types/options/read_only.rst.inc

```diff
@@ -1,3 +1,8 @@
+.. caution::
+
+    The ``read_only`` option has been deprecated and will be removed in 3.0.
+    Instead, use the ``attr`` option by setting it to an array with a ``readonly`` key.
```

Comment:
this block should be moved below the headline so that it fits inside the user's view port when the follow anchor refs.